definitions/CMakeLists.txt

```diff
@@ -35,6 +35,16 @@ set(WORKING_VARIABLE ${JSONRPC_PATTERNS})
 list(TRANSFORM WORKING_VARIABLE PREPEND "${CMAKE_SOURCE_DIR}/jsonrpc/")
 file(GLOB JSON_FILE ${WORKING_VARIABLE})
 
+if(NOT ENABLE_LEGACY_INTERFACE_SUPPORT)
```

Comment:
I hope we can just remove these json files from the repository... so no need to remove them only from the Cmake. If you remove them from the directory they will aslo not end up in the list..

Reply:
Ok, but actually I have added this flag to retain the json files to support older interfaces